Servers/UI/OJS.Servers.Ui/ClientApp/src/components/home-contests/contest-card/ContestCard.module.scss

```diff
@@ -49,3 +55,29 @@
   font-weight: bold;
   color: $text-light-gray-color;
 }
+
+.contestCardTitle {
+  white-space: nowrap;
+  overflow: hidden;
+  display: block;
+  text-overflow: ellipsis;
+  width: 225px;
```

Comment:
Try with `max-width`:
```
width: 100%;
max-width: 225px;
```


This will simplify the mobile version... one day...